lib/widgets/meal_item.dart

```diff
@@ -110,7 +110,7 @@ class MealItem extends StatelessWidget {
                         Icons.schedule,
                       ),
                       SizedBox(
-                        width: 6,
+                        width: 6
```

Comment:
please fix formatting, this should be aligned with SizedBox since it doesn't have trailing commas